### ISSUE TITLE
Support for additional urls

### ISSUE THREAD

**Issue by softmetz** [internetitem/logback-elasticsearch-appender#16]
In case of the selected ES node being down it would be handy to be able to specify one or more fallback URLs that are tried to publish the messages to.

**Comment by Xyaren** [comment]
I Agree with that. Definitely a must have for productive enviroments.

**Comment by abatkin** [comment]
Sounds great - feel free to send a PR if this interests you

**Comment by jzoe** [comment]
I Agree with that.

**Comment by SmiMi** [comment]
**I Agree with that**
